app/screens/settings/about/about.tsx

```diff
@@ -171,6 +185,14 @@ const About = ({componentId, config, license}: AboutProps) => {
         popTopScreen(componentId);
     });
 
+    const copyToClipboard = () => {
+        const copiedString = `${intl.formatMessage({id: 'settings.about.version', defaultMessage: 'App Version:'})} ${intl.formatMessage({id: serverVersion.id, defaultMessage: serverVersion.defaultMessage}, serverVersion.values)} \n ${intl.formatMessage({id: 'settings.about.server.version.desc', defaultMessage: 'Server Version:'})} ${intl.formatMessage({id: serverVersion.id, defaultMessage: serverVersion.defaultMessage}, serverVersion.values)} \n ${intl.formatMessage({id: 'settings.about.database', defaultMessage: 'Database:'})}  ${intl.formatMessage({id: 'settings.about.database.value', defaultMessage: `${config.SQLDriverName}`})} \n ${intl.formatMessage({id: 'settings.about.database.schema', defaultMessage: 'Database Schema Version:'})} ${intl.formatMessage({
```

Comment:
This line is hardly readable. What do you think about having all the strings generated on their own variables (or an object) and just use them here and later in the return? Something like...

```
const appVersionText = intl.formatMessage(...)
const serverVersionText = intl.formatMessage(...)
...

const copiedString = `${appVersionDesc} ${appVersion}
${serverVersionDesc} ${serverVersion}
...
`
```

Reply:
Yes I did that